src/plugin/persist.js

```diff
@@ -158,14 +158,21 @@ eg.module("persist", ["jQuery", eg, window, document], function($, ns, global, d
 
 	function setStateByKey(key, data) {
 		var beforeData = getState();
-		beforeData[key] = data;
+
+		if (data === null || typeof data === "undefined" || data === "") {
```

Comment:
You're right. There is no such case that calling setStateByKey method with data (null, "", undefined).
I' ll remove this logic.